packages/contracts/contracts/interpreters/ETHInterpreter.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity 0.5.8;
+pragma experimental "ABIEncoderV2";
+
+
+contract ETHInterpreter {
+
+  struct ETHTransferWrapper {
+    ETHTransfer[] wrapped;
```

Comment:
Why this versus `address[]` and `uint256[]`?

Reply:
`(address, uint256)[]` enforces that they are of the same length